README.md

```diff
@@ -40,20 +44,28 @@ The baseline for this dataset is Molecule Attention Transformer, fine-tuned with
 The baseline for this dataset is Audio Spectrogram Transformer, more details can be found in [paper](https://arxiv.org/abs/2104.01778).
 
 ## Question 1
-Can pretrained language models transfer to different modalities?
+* Can pretrained language models transfer to different modalities?
+* Only input embedding layer, output readout layer and layer norm parameters are fine-tuned.
```

Comment:
It' rather part of Empirical Results AFAIK.

Reply:
I afraid I don't get what you mean?

Reply:
You should introduce our "experimental setting" in the "Empirical Results" part. It's not a part of the question.